File: Beginner/1018 - Banknotes/main.py
value = int(input())

print(value)
print("%d nota(s) de R$ 100,00" % (value // 100))
value %= 100
print("%d nota(s) de R$ 50,00" % (value // 50))
value %= 50
print("%d nota(s) de R$ 20,00" % (value // 20))
value %= 20
print("%d nota(s) de R$ 10,00" % (value // 10))
value %= 10
print("%d nota(s) de R$ 5,00" % (value // 5))
value %= 5
print("%d nota(s) de R$ 2,00" % (value // 2))
value %= 2
print("%d nota(s) de R$ 1,00" % (value))

File: Beginner/1018 - Banknotes/main.cs
using System;
using System.Linq;

class URI
{
    static void Main(string[] args)
    { 
        int value_amount = Int32.Parse(System.Console.ReadLine().Trim());

        Console.WriteLine(value_amount);
        Console.WriteLine((value_amount / 100) + " nota(s) de R$ 100,00");
        value_amount %= 100;
        Console.WriteLine((value_amount / 50) + " nota(s) de R$ 50,00");
        value_amount %= 50;
        Console.WriteLine((value_amount / 20) + " nota(s) de R$ 20,00");
        value_amount %= 20;
        Console.WriteLine((value_amount / 10) + " nota(s) de R$ 10,00");
        value_amount %= 10;
        Console.WriteLine((value_amount / 5) + " nota(s) de R$ 5,00");
        value_amount %= 5;
        Console.WriteLine((value_amount / 2) + " nota(s) de R$ 2,00");
        value_amount %= 2;
        Console.WriteLine(value_amount + " nota(s) de R$ 1,00");
    }
}